qcodes/instrument_drivers/oxford/MercuryiPS_VISA.py

```diff
@@ -299,8 +299,89 @@ def __init__(self, name: str, address: str, visalib=None,
                            get_cmd=self._get_ramp_rate,
                            set_cmd=self._set_ramp_rate)
 
+
+        self.add_parameter('theta_saferamp',
+						   label='Theta',
+						   unit='degrees',
+						   set_cmd=self._theta_saferamp_set,
+						   get_cmd=self._theta_saferamp_get)
+		
+        self.add_parameter('r_saferamp',
+						   label='B_Radius',
+						   unit='T',
+						   set_cmd=self._r_saferamp_set,
+						   get_cmd=self._r_saferamp_get)
+
+        self.add_parameter('r_simulramp',
```

Comment:
shall there be also `phi_simulramp` and `theta_simulramp`? Also, shall there be corresponding parameters for x/y/z/rho?